.circleci/config.yml

```diff
@@ -14,10 +14,15 @@ jobs:
       - checkout
       - run: git submodule sync --recursive
       - run: git submodule update --recursive --init
+      - run: curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
[Rox requires that Rust be available at compile time](https://github.com/urbint/rox#dependencies)